components/central-application-gateway/internal/proxy/reverseproxy.go

```diff
@@ -150,9 +190,9 @@ func urlRewriter(gatewayURL, target, loc *url.URL) *url.URL {
 }
 
 func codeRewriter(rw http.ResponseWriter, err error) {
```

Comment:
Doest it make sense to keep this method? It's used only once, and `func responseModifier` is also changing the status code without using this method.